chef/roles/nova-multi-compute-zvm.rb

```diff
@@ -0,0 +1,7 @@
+name "nova-multi-compute-zvm"
+description "Installs requirements to run a Compute node in a Nova cluster"
+run_list(
+         "recipe[nova::zvm]",
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.<br>Style/FirstParameterIndentation: Indent the first parameter one step more than the previous line.